impala/assets/dashboards/impala_catalog_overview.json

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
Great job on the dashboards! My only note is that you should have more template variables like host, endpoint, and $scope.

Reply:
Thanks! Added host and scope to the three service overview dashboards because I dont think they would make sense to the global one. I did not add the openmetrics one because we can not filter the logs using this one since they do not have it

Reply:
I think it still makes sense to have it on the global one because what if a customer has multiple impala services?

Reply:
Adding it!